rosidl_typesupport_connext_cpp/cmake/rosidl_typesupport_connext_cpp_generate_interfaces.cmake

```diff
@@ -61,11 +61,21 @@ endforeach()
 
 # If not on Windows, disable some warnings with Connext's generated code
 if(NOT WIN32)
-  set(_connext_compile_flags
-    "-Wno-tautological-compare"
-    "-Wno-return-type-c-linkage"
-    "-Wno-deprecated-register"
-  )
+  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+    set(_connext_compile_flags
+      "-Wno-deprecated-register "
+      "-Wno-mismatched-tags "
+      "-Wno-return-type-c-linkage "
+      "-Wno-sometimes-uninitialized "
+      "-Wno-tautological-compare "
+      "-Wno-unused-variable "
```

Comment:
The `set` command creates a list here. This implies that the items are separated by a semicolon. Therefore each item should not have a trailing space.

The additional problem here is that the replace call in line 79 has a bug and does not what it is supposed to do, namely replace these semicolons with space. That is because the value of the replace call is not quoted. It should be:

    string(REPLACE ";" " " _connext_compile_flags "${_connext_compile_flags}")